packages/flutter/test/widgets/slivers_keepalive_test.dart

```diff
@@ -9,9 +9,9 @@ import 'package:flutter/rendering.dart';
 void main() {
   testWidgets('Sliver with keep alive without key - should dispose after reodering', (WidgetTester tester) async {
     List<Widget> childList= <Widget>[
-      WidgetTest0(text: 'child 0', keepAlive: true),
-      WidgetTest1(text: 'child 1', keepAlive: true),
-      WidgetTest2(text: 'child 2', keepAlive: true),
+      const WidgetTest0(text: 'child 0', keepAlive: true),
```

Comment:
here and below: Shouldn't the entire list be marked as const?

Reply:
I don't think so...  `childList` gets reassigned below. See:

```dart
childList = createSwitchedChildList(childList, 0, 2);
```

Reply:
Why does it matter that childList gets reassigned? My understanding is that the following is valid dart code, no?

```dart
    List<Widget> childList= const <Widget>[
      WidgetTest0(text: 'child 0', keepAlive: true),
      WidgetTest1(text: 'child 1', keepAlive: true),
      WidgetTest2(text: 'child 2', keepAlive: true),
    ];
   childList = createSwitchedChildList(childList, 0, 2);
```

Reply:
Hm, but `createSwitchedChildList` actually modifies the list. That seems kinda dodgy since it is called "create...". Oh well.

Reply:
I actually missed that too.  It's a little confusing; agreed...